src/components/navbar/navbar.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Navbar
+ * Contains a header that can display: Brand logo and list of
+ * linked items. Also is mobile-friendly
+ *
+ * author: George Gelinas <gelinas.george@gmail.com>
+ */
+
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import './navbar.css'
+
+const Navbar = ({ logo, items, isMobile }) => {
+  const [isDrawerOpen, setDrawerToggle] = useState(false)
+  return (
+    <>
+      <header className='header'>
+        <div className='brand-logo'>
+          <a href='#'>
+            <img src={logo} alt='Logo' />
+          </a>
+        </div>
+      </header>
+      {isMobile && (
+        <div className='hamburger-menu'>
+          <button
+            onChange={() => setDrawerToggle(!isDrawerOpen)}
+            className='hamburger-menu-cta'
+          >
+            {/* save the svg as its own file */}
```

Comment:
Still need to find a better SVG for the hamburger menu. Also to separate it out to its own file to source as an icon